opencti-platform/opencti-front/src/private/components/assets/devices/DeviceCreation.js

```diff
@@ -133,13 +133,17 @@ class DeviceCreation extends Component {
   }
 
   onSubmit(values, { setSubmitting, resetForm }) {
-    // const finalValues = pipe(
-    //   assoc('createdBy', values.createdBy?.value),
-    //   assoc('objectMarking', pluck('value', values.objectMarking)),
-    //   assoc('objectLabel', pluck('value', values.objectLabel)),
-    // )(values);
-    // CM(environmentDarkLight, {
-      // mutation: deviceCreationMutation,
+    const ports = {
+      "port_number": values.port_number,
+      "protocols": values.protocols || 'NetBIOS',
```

Comment:
why is 'NetBIOS' used?  If you're looking to set a default, a much better choice would be 'TCP'